### PR TITLE
0.9.2 (#64)

### PR DESCRIPTION
**Pull request by ericgaspar**
* Apply example_ynh
* Upgrade to 0.9.2

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Review by yalh76 — CHANGES_REQUESTED** [review]
It seems not to be just a 0.9.2 upgrade as written in the first comment of this PR

**Comment by ericgaspar** [comment]
yes I could also say `apply example_ynh` but it wouldn't help...

File: scripts/upgrade [hunk 1]
```diff
@@ -136,66 +130,37 @@ ynh_add_fpm_config
...
-
-	ynh_add_config --template="../conf/baikal.yaml" --destination="$bk_conf"
-
-	#=================================================
-	# UPGRADE BAIKAL
-	#=================================================
-	ynh_script_progression --message="Upgrading Baïkal..."
-
-	# Run Baikal upgrade
-	php"${phpversion}" "${final_path}/bin/upgrade.sh"
```
Comment (yalh76) [inline scripts/upgrade:166]: There is no more need to execute the `/bin/upgrade.sh` script ?
If not, maybe files in `sources/extra_files/app` should be removed

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by yalh76** [comment]
> yes I could also say `apply example_ynh` but it wouldn't help...

It helps to understand what you are doing in the PR, and lets know that the review will be hudge

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**